packages/jest-types/__typetests__/globals.test.ts

```diff
@@ -268,18 +268,21 @@ expectType<void>(test.concurrent.failing(() => {}, asyncFn));
 expectType<void>(test.concurrent.failing(function named() {}, asyncFn));
 expectType<void>(test.concurrent.failing(class {}, asyncFn));
 expectType<void>(test.concurrent.failing(class Named {}, asyncFn));
+expectType<typeof test.concurrent.each>(test.concurrent.failing.each);
```

Comment:
Looks good. Just few style related changes to keep the test suite consistent.

```suggestion

expectType<typeof test.concurrent.each>(test.concurrent.failing.each);
```